packages/onboarding/src/intent-screen/test/index.tsx

```diff
@@ -68,58 +75,123 @@ const intentsAlt: SelectItemAlt< string >[] = [
-				.forEach( ( button ) => button.simulate( 'click' ) );
-			expect( onSelect.mock.calls ).toEqual( [ [ 'value' ], [ 'value' ] ] );
-			expect( wrapper.find( 'button.select-items__item-button' ).first().text() ).toBe(
-				'Action Text'
+		it( 'should have a working button', async () => {
+			const user = userEvent.setup();
+
+			render(
+				<IntentScreen
+					intents={ intents }
+					intentsAlt={ intentsAlt }
+					onSelect={ onSelect }
+					preventWidows={ preventWidow }
+				/>
 			);
+
+			const button = screen.getAllByRole( 'button', { name: 'Action Text' } )[ 0 ];
+
+			expect( button ).toBeVisible();
+
+			await user.click( button );
+
+			expect( onSelect ).toHaveBeenCalledTimes( 1 );
+			expect( onSelect ).toHaveBeenCalledWith( 'value' );
 		} );
 
 		it( 'should have an icon', () => {
-			expect( wrapper.find( '.select-items__item svg' ) ).toHaveLength( 2 );
+			const { container } = render(
+				<IntentScreen
+					intents={ intents }
+					intentsAlt={ intentsAlt }
+					onSelect={ onSelect }
+					preventWidows={ preventWidow }
+				/>
+			);
+
+			expect( container.querySelectorAll( '.select-items__item svg' ) ).toHaveLength( 2 );
 		} );
 	} );
 
 	describe( 'SelectItemAlt', () => {
 		it( 'should have a description', () => {
-			expect( wrapper.find( 'p.select-items-alt__item-description' ) ).toHaveLength( 2 );
+			const { container } = render(
+				<IntentScreen
+					intents={ intents }
+					intentsAlt={ intentsAlt }
+					onSelect={ onSelect }
+					preventWidows={ preventWidow }
+				/>
+			);
+
+			expect( container.querySelectorAll( 'p.select-items-alt__item-description' ) ).toHaveLength(
+				2
+			);
 		} );
 
-		it( 'should have a working button', () => {
-			wrapper
-				.find( 'button.select-items-alt__item-button' )
-				.forEach( ( alt ) => alt.simulate( 'click' ) );
-			expect( onSelect.mock.calls ).toEqual( [ [ 'value-alt' ] ] );
-			expect( wrapper.find( 'button.select-items-alt__item-button' ).first().text() ).toBe(
-				'Action Text Alt'
+		it( 'should have a working button', async () => {
+			const user = userEvent.setup();
+
+			render(
+				<IntentScreen
+					intents={ intents }
+					intentsAlt={ intentsAlt }
+					onSelect={ onSelect }
+					preventWidows={ preventWidow }
+				/>
 			);
+
+			const button = screen.getAllByRole( 'button', { name: 'Action Text Alt' } )[ 0 ];
+
+			expect( button ).toBeVisible();
+
+			await user.click( button );
+
+			expect( onSelect ).toHaveBeenCalledTimes( 1 );
+			expect( onSelect ).toHaveBeenCalledWith( 'value-alt' );
 		} );
 
 		it( 'able to be hidden', () => {
-			expect( wrapper.find( '.select-items-alt__item' ) ).toHaveLength( 2 );
+			const { container } = render(
+				<IntentScreen
+					intents={ intents }
+					intentsAlt={ intentsAlt }
+					onSelect={ onSelect }
+					preventWidows={ preventWidow }
+				/>
+			);
+
+			expect( container.querySelectorAll( '.select-items-alt__item' ) ).toHaveLength( 2 );
 		} );
 
 		it( 'able to be disabled', () => {
```

Comment:
Missing 'should' too

Reply:
Done as part of 389083fd0e8a2e4c9fa10bf2a26bb87cac295c29.